frontend/src/containers/NavigationTree.js

```diff
@@ -353,24 +356,33 @@ class NavigationTree extends Component {
         </div>
 
         {/* sitemap items are listed using this */}
-        <div className="column-wrapper">
-          <div className="sitemap-column-left">
-            {sitemapLeftColItems &&
-              sitemapLeftColItems.map((subitem, subindex) =>
-                this.renderMenuOverlayContainer(subitem, subindex)
-              )}
-          </div>
-          <div className="sitemap-column-right">
-            {sitemapRightColItems &&
-              sitemapRightColItems.map((subitem, subindex) =>
-                this.renderMenuOverlayContainer(subitem, subindex)
-              )}
+        {!pendingQuery && (
+          <div className="column-wrapper">
+            <div className="sitemap-column-left">
+              {sitemapLeftColItems &&
+                sitemapLeftColItems.map((subitem, subindex) =>
+                  this.renderMenuOverlayContainer(subitem, subindex)
+                )}
+            </div>
+            <div className="sitemap-column-right">
+              {sitemapRightColItems &&
+                sitemapRightColItems.map((subitem, subindex) =>
+                  this.renderMenuOverlayContainer(subitem, subindex)
+                )}
+            </div>
+
+            {queriedResults.length === 0 && query !== '' && (
```

Comment:
IMHO the expression shall be `!pendingQuery && queriedResults.length === 0`

Reply:
Same case as for the MenuOverlay with the remark that here the no results message will be visible only for a split second  (as the tables with sitemap data are rendered visible). So in this case we also need to have query!== ''